chains/evm/voter/voter_test.go

```diff
@@ -59,6 +60,29 @@ func (s *VoterTestSuite) TestVoteProposal_HandleMessageError() {
 	s.NotNil(err)
 }
 
+func (s *VoterTestSuite) TestVoteProposal_SimulateVoteProposalError() {
+	s.mockMessageHandler.EXPECT().HandleMessage(gomock.Any()).Return(&proposal.Proposal{
+		Source:       0,
+		DepositNonce: 0,
+	}, nil)
+	s.mockClient.EXPECT().RelayerAddress().Return(common.Address{})
+	//calls.IsProposalVotedBy
+	s.mockClient.EXPECT().CallContract(gomock.Any(), gomock.Any(), gomock.Any()).Return(proposalNotVotedResponse, nil)
+	//calls.ProposalStatus
+	s.mockClient.EXPECT().CallContract(gomock.Any(), gomock.Any(), gomock.Any()).Return(inactiveProposalStatus, nil)
+	//calls.GetThreshold
+	s.mockClient.EXPECT().CallContract(gomock.Any(), gomock.Any(), gomock.Any()).Return(threshold, nil)
+
+	//calls.SimulateVoteProposal called 6 times
+	s.mockClient.EXPECT().CallContract(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil, errors.New("error"))
```

Comment:
You can set this only once and specify how many times should it be called.
Your tests are failing on CI anyway